doc/source/whatsnew/v0.9.1.rst

```diff
@@ -153,9 +149,10 @@ API changes
 
     .. ipython:: python
 
-        data = 'A,B,C\n00001,001,5\n00002,002,6'
+        from pandas.compat import StringIO
```

Comment:
can you import this from `io`? (`from io import StringIO)`

Reply:
I can do that. But it will be a deviation from most of the documents where `StringIO` comes from `pandas.compat`.

Reply:
1. categorical.rst
2. io.rst
3. cookbook.rst
4. whatsnew/ 19.0, 20.0, 21.0, 24.0, 

All the above import from `pandas.compat`. Confirm if I should change to `from io import StringIO`

Reply:
Everywhere in the pandas documentation (at least the one for users, not for contributors) should use `io` and not `compat`. We use `compat` ourself because pandas has to work in both Python 2 and 3, but users will run just one verison, and will use `io`. And I guess we may get rid of `compat` soon, as we're dropping support of Python 2.

So, you can replace all the cases if you want (and even we could probably add to `ci/code_checks.py` a `grep` to make sure we don't use it in any `doc/*.rst` file.

Reply:
Btw, my preferred option would be `import io` and `io.StringIO`. But also happy with `from io import StringIO`, which will be faster to change.